tiledb/sm/storage_manager/storage_manager.cc

```diff
@@ -1981,22 +2012,26 @@ Status StorageManager::load_array_schema(
   return Status::Ok();
 }
 
-Status StorageManager::load_all_array_schemas(
-    const URI& array_uri,
-    const EncryptionKey& encryption_key,
-    std::unordered_map<std::string, tdb_shared_ptr<ArraySchema>>&
-        array_schemas) {
+std::tuple<
+    Status,
+    std::optional<std::unordered_map<std::string, tdb_shared_ptr<ArraySchema>>>>
+StorageManager::load_all_array_schemas(
+    const URI& array_uri, const EncryptionKey& encryption_key) {
   auto timer_se = stats_->start_timer("read_load_all_array_schemas");
+  std::unordered_map<std::string, tdb_shared_ptr<ArraySchema>> array_schemas;
```

Comment:
Would be clearer to move the declaration of this variable down to the bottom where it's initialized.

Reply:
Done